src/SupportMessage/SupportMessage.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { SupportMessage, SupportMessageProps } from './SupportMessage'
+import { CollectionPage } from './Collection'
+
+export default {
+  title: 'SupportMessage',
+  component: SupportMessage,
+}
+
+const Template = (props: SupportMessageProps) => (
+  <SupportMessage {...props}>Support message for customer</SupportMessage>
+)
+
+export const Default = Template.bind({})
```

Comment:
you need to bing the args of the default template if you want to make it interactive